mcs/tools/resx2sr/resx2sr.cs

```diff
@@ -40,7 +40,6 @@ class CmdOptions
 	{
 		public bool ShowHelp { get; set; }
 		public string OutputFile { get; set; }
-		public bool ExistingOnly { get; set; }
```

Comment:
what's the reason for the resx2sr.cs changes?

Reply:
This is now https://github.com/mono/mono/pull/11951.  Since the tool got unusably broken in that commit, it will have to be reverted.